src/components/resource/edit-managed-title/resource-edit-managed-title.js

```diff
@@ -162,26 +162,38 @@ class ResourceEditManagedTitle extends Component {
               <DetailsViewSection
                 label="Coverage dates"
               >
-                <CustomCoverageFields
-                  initialValue={initialValues.customCoverages}
-                />
+                {managedResourceSelected ? (
+                  <CustomCoverageFields
+                    initialValue={initialValues.customCoverages}
+                  />
+                ) : (
+                  <p>Add the package to holdings to set custom coverage dates.</p>
```

Comment:
Text for all of these should be `Add the resource to holdings` instead of `package`.

Reply:
So right!